ci/gh-actions-security.md

```diff
@@ -0,0 +1,78 @@
+# GitHub Actions and Security
+
+We heavily leverage GitHub Actions not just for CI testing but do execute code to accomplish a wide variety of Ponylang organization tasks.
+
+Most of the executed code is created and published by either us or GitHub. We also use some actions that are published by additional 3rd parties.
+
+## Our Rules for GitHub Actions
+
+- PRs are not allowed to run actions without approval unless the person opening the PR is a member of the Ponylang organization
+- Only actions from GitHub's `actions` organization and our own organization can run using a Docker image or references a git branch or tag
+- All "additional 3rd party" actions **MUST** only for a specific git commit hash
+- Only "additional 3rd party" actions that have been reviewed and considered trustworthy can be used
+- Only Docker images that have been blessed by DockerHub or GitHub can be used as base images for images that we use as part of our CI process if the image is referenced by a tag
```

Comment:
Would be nice to be more explicit about what "blessed" means here.

Reply:
"vouched for by"?

I'm not sure what the proper term should be.

Reply:
```suggestion
- Only Docker images that have been marked by DockerHub or GitHub as Official Images can be used as base images for images that we use as part of our CI process if the image is referenced by a tag
```